controller/volume_controller_test.go

```diff
@@ -847,6 +847,12 @@ func (s *TestSuite) runTestCases(c *C, testCases map[string]*VolumeTestCase) {
 		c.Assert(err, IsNil)
 		pIndexer.Add(p)
 
+		ei, err := lhClient.LonghornV1alpha1().EngineImages(TestNamespace).Create(newEngineImage(types.EngineImageStateReady))
```

Comment:
Why we need those?

Reply:
`IsEngineImageCLIAPIVersionOne()` is used in volume controller and it needs the engine image to check the version